### PR TITLE
RWA-3563: Fixed Sonar issues

### PR DESCRIPTION
**Pull request by adityadwadasi**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RWA-3563 

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceCustomRepositoryImpl.java [hunk 1]
```diff
@@ -103,7 +105,7 @@ private String extraConstraints(List<String> excludeCaseIds, SearchRequest searc
         if (searchRequest.isAvailableTasksOnly()) {
             extraConstraints.append("AND assignee IS NULL ");
         } else {
-            extraConstraints.append(buildListConstraint(searchRequest.getUsers(), "assignee", "assignee", true));
+            extraConstraints.append(buildListConstraint(searchRequest.getUsers(), ASSIGNEE, ASSIGNEE, true));
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceCustomRepositoryImpl.java:108]: Can I suggest a convention for naming some of these?  
Maybe if it's a DB column name we can call it something like `COL_ASSIGNEE` or `DB_COL_ASSIGNEE`?

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java [hunk 2]
```diff
@@ -112,8 +115,8 @@ public Map<String, CamundaVariable> getTaskVariables(String taskId) {
 
     public boolean isCftTaskStateExistInCamunda(String taskId) {
         Map<String, Object> body = Map.of(
-            "variableName", CFT_TASK_STATE.value(),
-            "taskIdIn", singleton(taskId)
+            VARIABLE_NAME, CFT_TASK_STATE.value(),
+            TASK_ID_IN, singleton(taskId)
```
Comment (raghera) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java:119]: Maybe we can have a similar convention here e.g. `KEY_VARIABLE_NAME` etc

Comment (adityadwadasi) [reply src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java]: I made the changes as suggested Ravi.

**Review by raghera — DISMISSED** [review]

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/restrict/ClientAccessControlService.java [hunk 3]
```diff
@@ -14,6 +14,8 @@
 @Service
 public class ClientAccessControlService {
 
+    public static final String SRV_AUTH_MUST_NOT_BE_NULL = "ServiceAuthorization must not be null";
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/restrict/ClientAccessControlService.java:17]: Please refer to agreed naming convention for Constants -> https://hmcts-reform.slack.com/archives/C06DPL6U318/p1723714764215049

File: src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceCustomRepositoryImpl.java [hunk 4]
```diff
@@ -28,6 +28,8 @@ public class TaskResourceCustomRepositoryImpl implements TaskResourceCustomRepos
         + "%s%s%s";
 
     private static final String SELECT_CLAUSE = "SELECT t.task_id ";
+
+    private static final String DB_COL_ASSIGNEE = "assignee";
```
Comment (BenNzewi) [inline src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/repository/TaskResourceCustomRepositoryImpl.java:32]: Please refer to agreed naming convention for Constants -> https://hmcts-reform.slack.com/archives/C06DPL6U318/p1723714764215049

**Review by BenNzewi — APPROVED** [review]

**Pull request merged**